### PR TITLE
[deckhouse] Ability to send update notifications for Deckhouse patch releases

### PR DESCRIPTION
**Pull request by libmonsoon-dev**
## Why do we need it, and what problem does it solve?
Close https://github.com/deckhouse/deckhouse/issues/8878

## Checklist
- [x] The code is covered by unit tests.
- [x] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: deckhouse-controller
type: feature
summary: Added ability to send update notifications for Deckhouse patch releases
impact: The format of webhook notifications about updates has been changed: the value in the version field has been changed from "x.y" to "x.y.z"
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @libmonsoon-dev. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @libmonsoon-dev. I've started the workflow for label 'e2e/run/static'...

File: modules/002-deckhouse/openapi/doc-ru-config-values.yaml [hunk 1]
```diff
@@ -124,6 +124,11 @@ properties:
                     Токен для авторизации на webhook.
 
                     Токен будет в заголовке `Authorization` в формате `Bearer <token>`.
+          releaseType:
+            description: |
+              releaseType определяет на какие версии будет отправлена нотификация:
+              `Minor`: только для релизов с новой минорной версией
+              `All`: для всех релизов включая патч версии
```
Comment (z9r5) [inline modules/002-deckhouse/openapi/doc-ru-config-values.yaml:131]:
```suggestion
              Определяет тип версии, о которой будет отправлено оповещение:
              - `Minor` — только для обновления минорной версии.
              - `All` — для любых обновлений, включая обновление патч версии.
```

File: modules/002-deckhouse/openapi/config-values.yaml [hunk 2]
```diff
@@ -206,6 +206,17 @@ properties:
                     The token for the webhook.
 
                     The token will be sent in the `Authorization` header in the format `Bearer <token>`.
+          releaseType:
+            type: string
+            description: |
+              releaseType defines which versions the notification will be sent to:
+              `Minor`: only for releases with a new minor version
+              `All`: for all releases including patch versions
```
Comment (z9r5) [inline modules/002-deckhouse/openapi/config-values.yaml:214]:
```suggestion
              Defines the type of version for which the notification will be sent:
              - `Minor` — only for updating the minor version.
              - `All` — for any updates, including updating a patch version.
```

**Review by z9r5 — APPROVED** [review]

File: modules/002-deckhouse/openapi/doc-ru-config-values.yaml [hunk 3]
```diff
@@ -124,6 +124,11 @@ properties:
                     Токен для авторизации на webhook.
 
                     Токен будет в заголовке `Authorization` в формате `Bearer <token>`.
+          releaseType:
+            description: |
+              Определяет тип версии, о которой будет отправлено оповещение:
+              - `Minor` — только для обновления минорной версии.
+              - `All` — для любых обновлений, включая обновление патч версии.
```
Comment (z9r5) [inline modules/002-deckhouse/openapi/doc-ru-config-values.yaml:131]:
```suggestion
              - `All` — для любых обновлений, включая обновление патч-версии.
```

**Review by yalosev — APPROVED** [review]

**Pull request merged**